Released version 1.1.0 of unittest2 hardcodes a requirement for the `argparse`
package. We (conda-forge) don't provide such a package for Python 2.7 since it
became built-in, so we have to remove the dependency. This requirement has
since been fixed in unittest2
(https://hg.python.org/unittest2/rev/d091f0086b03).

--- a/setup.py
+++ b/setup.py
@@ -57,7 +57,7 @@
 # Both install and setup requires - because we read VERSION from within the
 # package, and the package also exports all the APIs.
 # six for compat helpers
-REQUIRES = ['argparse', 'six>=1.4', 'traceback2'],
+REQUIRES = ['six>=1.4', 'traceback2'],
 
 params = dict(
     name=NAME,
